docs/modules/reference/pages/glossary.adoc

```diff
@@ -0,0 +1,25 @@
+
+= Glossary
+
+This glossary defines terms that are specific to the OpenNMS ALEC plugin.
+For more general terminology related to the OpenNMS project, see the https://docs.opennms.com/horizon/latest/reference/glossary.html[core documentation].
+
+== A
+
+alarm:: A fault or problem that is triggered by one or more events.
+
+ALEC:: Architecture for Learning Enabled Correlation.
+
+== C
+
+cluster:: A group of correlated alarms.
+
+== I
+
+inventory object (IO):: An abstraction of monitored nodes and their components.
+Alarms can be related to a minimum of zero inventory objects, and a maximum of one.
+
+== S
+
+situation:: A representation of a synthetic "parent" alarm which is triggered by one or more "child" alarms.
+Situations should only be created if there are two or more alarms in an alarm causality tree; otherwise, the event could be represented by a single alarm.
```

Comment:
```suggestion
Situations should be created only if there are two or more alarms in an alarm causality tree; otherwise, the event could be represented by a single alarm.
```